plugins/modules/aci_l3out_bfd_interface_profile.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain (@anvjain)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_bfd_interface_profile
+short_description: Manage L3Out BFD Interface profile.
+description:
+- Manage L3Out BFD Interface profile (bfdIfP) configuration on Cisco ACI fabrics.
+- Only available in APIC version 5.2 or later and for non-cloud APICs.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  l3out_logical_node_profile:
+    description:
+    - Name of an existing L3Out Logical Node profile.
+    type: str
+    aliases: [ logical_node_profile, logical_node_profile_name ]
+  l3out_logical_interface_profile:
+    description:
+    - Name of an existing L3Out Logical Interface profile.
+    type: str
+    aliases: [ logical_interface_profile, logical_interface_profile_name ]
+  name:
+    description:
+    - Name of the L3Out BFD Interface profile object.
+    type: str
+    aliases: [ bfd_multihop_interface_profile ]
+  name_alias:
+    description:
+    - Name Alias of the L3Out BFD Interface profile object.
+    type: str
+  description:
+    description:
+    - Description of the L3Out BFD Interface profile object.
+    type: str
+    aliases: [ descr ]
+  interface_profile_type:
+    description:
+    - Authentication Type of the L3Out BFD Interface profile object.
+    - APIC sets the default value to none.
+    type: str
+    choices: [ none, sha1 ]
+  key:
+    description:
+    - Authentication Key of the L3Out BFD Interface profile object.
+    type: str
+  key_id:
+    description:
+    - Authentication Key ID of the L3Out BFD Interface profile object.
+    - APIC sets the default value to 3.
+    - Allowed range is 1-255
+    type: int
+  bfd_interface_policy:
```

Comment:
for those attributes should we document that "" will delete the policy?

Reply:
setting the argument to "" will not exactly delete the policy it just sets the nome to "" and sets the value to default. hence adding it will delete the policy may confuse the user.